po/fr.po

```diff
@@ -201,10 +201,10 @@ msgid ""
 "\n"
 "<b>All the data in the drive will be erased</b>"
 msgstr ""
-"Le disque sélectionné doit être formaté pour être utilisé pour des "
```

Comment:
Should we respect the format @rastersoft ?

Reply:
and why does we have two time %s, one time for disque name, and one time for disk size?

Reply:
Because the best way of putting the disk identifier and the size in one language maybe is not the best in another one. This way, I give more freedom to the translators.

Reply:
yeh, but spontalty, I would think we have two different "%string". For example "%n" for the name and "%s" for the size

Reply:
The size is also an string because it also has the units (GB, MB...) with it. The true problem is that you always have to put the id first and the size after... I'm not sure if Vala supports numbered parameters like python... I'll check.

Reply:
Ok, I found how to solve it. I uploaded a change.

Reply:
and so ed37bfd do it for French